test/e2e/create_test.go

```diff
@@ -471,33 +471,6 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, expectError, checkVe
 		}
 
 		for _, node := range nodes {
-			// check node status
```

Comment:
Ready != schedulable

Reply:
I think you've renamed the function since this comment was originally left. The point still stands. We need to check for ready AND schedulable in more than just the initial create suite.